src/main/java/pl/nogacz/chess/board/Board.java

```diff
@@ -181,6 +197,166 @@ public void readMouseEvent(MouseEvent event) {
         }
     }
 
+    public void playerVsPlayer(MouseEvent event){
+        if(isWhitesTurn && !isBlacksTurn){
+            if(isBlacksTurn || isGameEnd) {
+                return;
+            }
+
+            Coordinates eventCoordinates = new Coordinates((int) ((event.getX() - 37) / 84), (int) ((event.getY() - 37) / 84));
+
+            if(!eventCoordinates.isValid()) {
+                return;
+            }
+
+            gameLogic.prepareData();
+
+            if(!gameLogic.isMovePossible()) {

```

Comment:
Too many if else

Refactor this code. Extract some of them into separate methods with proper name to improve comprehension

Reply:
Farklı metodlara böldüm ve daha modüler bir hale getirmeye çalıştım